### DISCUSSION TITLE
MAME on QT6

### DISCUSSION THREAD

**Discussion by XargonWan** [mamedev/discussions#81] [Q&A] [answered]
Greetings, I'm Xargon from RetroDECK.

We wish to include MAME on RetroDECK however our flatpak is working on QT6 framework.
Is it possible to compile MAME using QT6 instead of QT5?

Thanks in advance.

**Comment by cuavas** [comment]
It supposedly is possible to build against Qt6 on Linux, although it isn’t the default way to build MAME.  See mamedev/mame#11413 and mamedev/mame@4649cf024839d90b5ac202652367ede62d2dacd4.

**Reply by mmicko** [reply]
@XargonWan in case you actually do not need to have QT debugger available in your build, you can build it with  `USE_QTDEBUG=0 ` parameter and then QT libraries would not be needed at all.

**Reply by cuavas** [reply]
We’d really rather people don’t do this, though.  The debugger is a key feature of MAME, and the multi-window Qt debugger is much more usable than the minimal imgui debugger.

**Reply by XargonWan** [reply]
I see, well actually we don't expect that our users will debug MAME on RetroDECK as it's a "console experience", we even preconfigure it as we do with any other emulator.
Until I discover where the `QT_HOME` should point in the flatpak environment I can try to see if I can build it with that option and then in any case switch back to the `QT_HOME` approach.

**Reply by rb6502** [reply]
I didn't know any libretro versions of MAME even included the debugger.

**Reply by XargonWan** [reply, marked as answer]
> @XargonWan in case you actually do not need to have QT debugger available in your build, you can build it with `USE_QTDEBUG=0 ` parameter and then QT libraries would not be needed at all.

@mmicko  Actually this worked: it can compile, now I have to test it :)
